### DISCUSSION TITLE
Text field (text area) in list view is not truncated

### DISCUSSION THREAD

**Discussion by ArniSamano** [YetiForceCompany/YetiForceCRM#16485] [Feature improvement]
Field text area with load of characters goes full length in list view instead of being truncated (custom module)

![image](https://user-images.githubusercontent.com/105257433/190815918-9feec12e-45da-41a7-9ce3-e9d39ed7523a.png)

**Comment by ArniSamano** [comment]
My bad. This is configurable in field editor O.o

**Reply by apcloic** [reply]
@ArniSamano 
Remember to mark the discussion as Answered. 🙏😉

**Comment by ArniSamano** [comment]
Future Improvement don't have that button ;-)
